README.md

```diff
@@ -244,6 +264,18 @@ async deleteEvent(id: string, type: string): Promise<Event> {
 }
 ```
 
+#### Hierarchical Partition Keys
+
+If using hierarchical partition keys, you need to provide the partition key as an array of strings when calling one of the CRUD methods on the `Container`. For example, when reading a single resource:
```

Comment:
Apart from string other types are alo supported here is the defition of partition key from SDK.

``` 
/**
 * PartitionKey of a container.
 */
export type PartitionKey = PrimitivePartitionKeyValue | PrimitivePartitionKeyValue[];

/**
 * A primitive Partition Key value.
 */
export type PrimitivePartitionKeyValue =
  | string
  | number
  | boolean
  | NullPartitionKeyType
  | NonePartitionKeyType;
```